tools/config-generator/templates/prow/prow_config.yaml

```diff
@@ -1,152 +0,0 @@
-plank:
```

Comment:
Did you delete `tools/config-generator/templates/prow/prow_plugins.yaml` as well? I don't see it in the changed files.

Reply:
And also the generated config files for Prow and Prow staging?

Reply:
deleted in latest commit